verifyincluded.go

```diff
@@ -70,7 +58,7 @@ func verifyEvent(
 	// includes the event. Future work can deepen this to include
 	// discovery of the log head, and or verification against a
 	// sealed MMRSize.
-	verified, err := mmr.VerifyInclusion(&massif, sha256.New(), mmrSize, event.LeafHash, mmrIndex, proof)
+	verified, err := mmr.VerifyInclusion(&massif, sha256.New(), mmrSize, mmrEntry, mmrIndex, proof)
```

Comment:
Minor: Is there a reason why this code doesn't use the new methods on the AppEntry type?

Reply:
see reply on comment above :)